scheduler/scheduler.go

```diff
@@ -176,7 +177,10 @@ func (scheduler *Scheduler) SystemStart() (err error) {
 		common.LogDebug(common.Initialization, "Start STOP at scheduler", core, "core")
 	}
 	go func() {
-		low.SetAffinity(uint8(core))
+		err := low.SetAffinity(uint8(core))
+		if err != nil {
+			common.LogFatal(common.Initialization, "Failed to set affinity to", core, "core: ", err)
```

Comment:
maybe return common.WrapWithNFError instead of fatal logging?

Reply:
here and below

Reply:
I place LogFatal only in goroutines, because we do not return values from goroutines

Reply:
Here are several goroutines, each can give this error, maybe we should think of writing its errors to global channel or err variable?

Reply:
LogFatal only in goroutines -> it isn't right logic. Right logic is only via execution. Here is not execution - it is initialization of stop goroutine. So you can return error (one more time our errors are awful) with comment about "for stopping thread"

Reply:
So we need to return value from running goroutine. It seems correct way is to use channel https://www.atatus.com/blog/goroutines-error-handling/